src/accessibility/KeyboardShortcuts.ts

```diff
@@ -392,6 +395,13 @@ export const KEYBOARD_SHORTCUTS: IKeyboardShortcuts = {
         },
         displayName: _td("Navigate to previous message in composer history"),
     },
+    [KeyBindingAction.ShowStickerPicker]: {
+        default: {
+            ctrlOrCmdKey: true,
+            key: Key.SEMICOLON,
+        },
+        displayName: _td("Send a sticker"),
+    },
```

Comment:
![image](https://user-images.githubusercontent.com/2403652/156883524-de705fdb-9e5c-41fd-ac41-925b9522fcbb.png)

Cmd+; is unlikely to cause confusion so is fine

But IMO a toggle functionality would be better here rather than only opening

Reply:
Fixed in 2e33b9c